src/components/thumbnail/class-work.tsx

```diff
@@ -2,184 +2,76 @@ import { inject, observer } from "mobx-react";
-                canvasContext="class-work" document={publication} scale={scale}
-                captionText={captionText}
-                onDocumentClick={this.handleDocumentClick} onDocumentDragStart={this.handleDocumentDragStart}
-                isStarred={pubStar} onDocumentStarClick={onDocumentStarClick} />
-            );
-          })}
-          </div>
+        <div className="header">{classWorkTab.label}</div>
+
+        {classWorkTab.sections.map(section => (
+          <DocumentsSection
+            key={section.type} tab={classWorkTab.tab} section={section}
+            stores={this.stores} scale={this.props.scale}
+            isExpanded={this.state.showSection.get(section.type)}
+            onToggleExpansion={this.handleToggleExpansion}
+            onDocumentClick={this.handleDocumentClick}
+            onDocumentDragStart={this.handleDocumentDragStart}
+            onDocumentStarClick={_handleDocumentStarClick} />
+        ))}
       </div>
     );
   }
 
-  private renderStarredDocuments = (publications: DocumentModelType[]) => {
-    const { user } = this.stores;
-    const { scale } = this.props;
-    const sectionTitle = "Starred";
-    const isExpanded = this.state.showStarredDocuments;
-    return (
-      <div className="section starred">
-        <CollapsibleSectionHeader
-          sectionTitle={sectionTitle} dataTestName="class-work-section"
-          isExpanded={isExpanded} onClick={this.handleStarredSectionClicked}/>
-
-        <div className={"list " + (isExpanded ? "shown" : "hidden")}>
-          {publications.map((publication) => {
-            const captionText = this.getPublicationCaptionText(publication, sectionTitle);
-            const pubStar = user.type === "teacher"
-                              ? !!publication.stars.find(star => star.uid === user.id && star.starred)
-                              : !!publication.stars.find(star => star.starred);
-            const onDocumentStarClick = user.type === "teacher" ? this.handleDocumentStarClick : undefined;
-            return (
-              pubStar
-                ? <ThumbnailDocumentItem
-                    key={publication.key} dataTestName="class-work-list-items"
-                    canvasContext="class-work" document={publication} scale={scale}
-                    captionText={captionText}
-                    onDocumentClick={this.handleDocumentClick} onDocumentDragStart={this.handleDocumentDragStart}
-                    isStarred={true} onDocumentStarClick={onDocumentStarClick} />
-                : null
-            );
-          })}
-          </div>
-      </div>
-    );
-  }
+  // private getPublicationCaptionText(publication: DocumentModelType, sectionTitle: string) {
+  //   const pubUser = this.stores.class.getUserById(publication.uid);
+  //   const userName = pubUser && pubUser.fullName || "";
+  //   return userName + "\n" + (sectionTitle.length ? (sectionTitle) : "");
+  // }
```

Comment:
Are we leaving this in as a reference for future work?

Reply:
This can be removed, it was something I did and Kirk saved it for me in the event I'd want/need it back but I should be good now.

Reply:
Then I'll remove it.